inspirehep/modules/disambiguation/api.py

```diff
@@ -37,38 +38,50 @@
 from inspirehep.modules.disambiguation.utils import open_file_in_folder
 
 
-def save_signatures_and_clusters():
-    """Save signatures and starting clusters to disk.
+def save_curated_signatures_and_input_clusters():
+    """Save curated signatures and input clusters to disk.
 
-    Saves two files to disk called (by default) ``clusters.json`` and ``signatures.jl``.
-    The former contains a dictionary that represents the starting clusters that are used
-    by ``BEARD`` as ground truth, while the latter contains one line per curated signature
-    present in INSPIRE.
+    Saves two files to disk called (by default) ``input_clusters.jsonl`` and
+    ``curated_signatures.jsonl``. The former contains one line per each cluster
+    initially present in INSPIRE, while the latter contains one line per each
+    curated signature that will be used as ground truth by ``BEARD``.
     """
-    clusters = defaultdict(list)
+    signatures_with_author = defaultdict(list)
+    signatures_without_author = []
```

Comment:
stupid question: do you really need these two different structures, that then get serialized in the same way? can't you simply assign to the key `None` when there is no author?

Reply:
But won't this create a single cluster with all signatures without an author smooshed together, while we want each to form a cluster on their own?

Reply:
good point. So it was stupid after all :)

Reply:
I guess we can move the `if` later, and iterate while writing to the file if `author_id` is `None`...